### ISSUE TITLE
Add Bearer token Authentication to the "Requests Get" connector

### ISSUE THREAD

**Issue by joshsgoldstein** [sidprobstein/swirl-search#80]

**Comment by sidprobstein** [comment]
This is checked in to 1.9 branch swirl.connectors.requestsget.py
To specify a bearer token use the format:

```
"connector": "RequestsGet",
...etc...
"credentials": "bearer=some-api-token"
```

**Comment by erikspears** [comment]
Verified this fix in 1.9.1 via testing the new Miro integration.  Bearer tokens should be good to go, thanks @sidprobstein !
Closing this one out.
